### DISCUSSION TITLE
Aliased column is not respected when creating calculated field

### DISCUSSION THREAD

**Discussion by tinoptilogic** [RevealBi/Reveal.Sdk#116] [General]
After creating an alias for a column on visualization, the same alias is not respected when creating new calculated field.

Is that by design? If yes would it be possible to have aliased field appear when creating calculated field?

Thanks

**Comment by TorreyBetts** [comment]
Yes, that sounds like by design as the alias is related to the field you dragged to the visualization and not the field itself. With that said, we added an event in v1.4.0 at the beginning of this year that allows you to customize those fields. Here's the [documentation about it](https://help.revealbi.io/web/release-notes#all-platforms-7).
